docs/docs/getting-started/5-nocode-esp32-deployment.md

```diff
@@ -0,0 +1,35 @@
+# No-Code ESP32 Deployment
+
+We built a tool for deploying your Groundlight detector on any supported ESP32 Camera Board without writing any code. This tool is available at https://code.groundlight.ai/groundlight-embedded-uploader/.
+
+## Easy Deployment
+
+This tool is designed to make it as easy as possible to deploy your Groundlight detector on an ESP32 Camera Board. You can deploy your detector in just a few clicks.
+
+1. Go to https://code.groundlight.ai/groundlight-embedded-uploader/.
```

Comment:
I'm not crazy about the name `groundlight-embedded-uploader`.  We don't need "groundlight" in the project-name/URL.  The name should probably include "esp32" instead of "embedded".  And "uploader" is okay, but might be more self-explanatory as "configurator" or "firmware-flasher" or something like that.

Reply:
esp32-firmware-configurator or esp32-detector-configurator ?